api/tests/Feature/SkillTest.php

```diff
@@ -179,6 +179,49 @@ public function test_create_skill()
             ->assertJsonFragment(['name' => $variables['skill']['name']]);
     }
 
+    /**
+     * Test that non-unique keys fail
+     *
+     * @return void
+     */
+    public function test_create_non_unique_skill_key()
+    {
```

Comment:
Question, why this casing, just nitpicking as camel-case seems to be our standard otherwise

Reply:
I just copy and pasted from a test earlier in the file 🤷‍♀️ 

https://github.com/GCTC-NTGC/gc-digital-talent/blob/e3a876d13236fe2e475b1bdd1f62fc914344ffa1/api/tests/Feature/SkillTest.php#L105

Reply:
Huh, oh weird, guess that file has its own convention going 🤷

Reply:
You could create a ticket to make our methods [PSR compliant](https://www.php-fig.org/psr/psr-1/#43-methods) 😅

Reply:
Done #8479